src/cider_nrepl/plugin.clj

```diff
@@ -28,8 +31,10 @@
 
     (when-not lein-version-ok?
       (lein/warn "Warning: cider-nrepl requires Leiningen 2.5.2 or greater."))
-    (when-not clojure-version-ok?
+    (when (or (not clojure-version-ok?) clojure-excluded?)
```

Comment:
No. This was deliberate. If Clojure has been excluded, then `clojure-version-ok?` will be computed using the default based on Leiningen's version, which should be OK, and `clojure-version-ok?` will presumably be `true`. If Clojure is not in the `:dependencies`, we do want to emit a warning here, so `(or (not true) true) -> (or false true) -> true` is appropriate.